### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/utils/pmpermit.py [hunk 1]
```diff
@@ -310,11 +310,13 @@ async def pm_spam_callback(_, c_q: CallbackQuery):
         if c_q.from_user.id == owner.id:
             await c_q.answer("Sorry, you can't click by yourself")
         else:
-            await userge.block_user(c_q.from_user.id)
             del pmCounter[c_q.from_user.id]
             user_dict = await userge.get_user_dict(c_q.from_user.id)
+            user_dict.update(
```
Comment (rking32) [inline userge/plugins/utils/pmpermit.py:315]: hey don't add this. not working

File: userge/plugins/utils/pmpermit.py [hunk 2]
```diff
@@ -326,6 +328,8 @@ async def pm_contact_callback(_, c_q: CallbackQuery):
             await c_q.answer("Sorry, you can't click by yourself")
         else:
             user_dict = await userge.get_user_dict(c_q.from_user.id)
+            user_dict.update(
```
Comment (rking32) [inline userge/plugins/utils/pmpermit.py:331]: this too

File: userge/plugins/utils/pmpermit.py (hunk 1 above)
Comment (Krishna-Singhal) [reply userge/plugins/utils/pmpermit.py:315]: but if noPmmessage having {chat} object than ?

Comment (rking32) [reply userge/plugins/utils/pmpermit.py:315]: no use of {chat} in pm

**Pull request merged**